index.html

```diff
@@ -817,49 +821,50 @@
+- This opens an editor where you can enter commands for each commit to be rebased. You can also reorder the list to change the order for applying these commits
+- **interactive rebase commands**
   - These commands determine how individual commits will be transferred to the new base.
   - **p, pick** = use commit
   - **r, reword** = use commit, but edit the commit message
   - **e, edit** = use commit, but stop for amending
-  - **s, squash** = use commit, but meld into previous commit (will ask what the commit message for the join will be)
+  - **s, squash** = use commit, but meld into previous commit (you will be prompted for the squashed commit message)
   - **f, fixup** = like "squash", but discard this commit's log message
   - **x, exec** = run command (the rest of the line) using shell
   - **d, drop** = remove commit (you can also remove the line instead of this)
-- Used mainly to clean up a messy history before merging a feature branch into master
+- Interactive rebase is often used to clean up a messy history before merging a feature branch into master
 
 .center[**Remember that you shouldn't do this on public commits (commits that have been pushed to remotes)**]
 
 ---
 class: middle, center
 # rebase -i illustrated
 
-![alt text](images/rebasei.svg "rebase")
+![alt text](images/rebasei.svg "interactive rebase")
 
 ---
 name: rbm
 #.center[**Rebase** vs **Merge**]
 - Both of these commands are designed to integrate changes from one branch into another branch—they just do it in very different ways
 - Merge
-  - Merging is nice because it’s a non-destructive operation. The existing branches are not changed in any way
-  - The feature branch will have an extraneous merge commit every time you need to incorporate upstream changes
-  - **`git pull`** default behaviour
+  - Merging is nice because it’s a non-destructive operation. The existing branches are not changed in any way.
+  - The feature branch will have an extraneous merge commit every time you decide to incorporate upstream changes
+  - If not used sparingly, merging can create a history that is very hard to follow; it may also accidentally introduce unwanted commits in the history (the *spaghetti merge* problem).
+  - Merging is the default behavior for **`git pull`**
 - Rebase
   - Much cleaner & linear project history
-  - Loses the context provided by a merge commit—you can’t see when upstream changes were incorporated into the feature
-  - Can be catastrophic if done on public branches & history
+  - Rebasing loses the context provided by a merge commit—you won’t see how (when) distinct sets of changes were brought together
+  - Rebasing can be catastrophic if done on public branches & history
   - **`git pull --rebase`** to *unlock* this feat or **`git config --global branch.autosetuprebase always`**
 
 ---
 class: middle, center
-# destructive rebase illustrated
+# destructive rebase illustrated<br>.center.remark-code[master branch rebased on feature]
```

Comment:
I made a change to the svg so I think this is not needed anymore

Reply:
OK